manifests/globals.pp

```diff
@@ -0,0 +1,10 @@
+#
+# Global apache configuration.
+#
+class apache::globals(
+  Boolean $scl            = false,
```

Comment:
I think there shouldn't be a single SCL parameter. You want one for httpd and PHP. You could solve this by making ```scl_httpd``` an ```Optional[String]``` where undef is the current behavior of no SCL.

Reply:
I agree that it is better to make scl_httpd and scl_php_version optional, such that the user has to make a thought-through choice.

As for not having a scl-version, I think that when SCL is used, it is more like a separate distro, than just an updated package, for most everything changes: package names, service names, location on disk, modules etc.